about.html

```diff
@@ -0,0 +1,141 @@
+      <div class="showcase container row gap-4 col-12">
+        <div class="conference-container container-fluid text-light col-12 col-md-6" style="background-image: url(Images/recap1.jpg);">
+          <div class="conference-bg container-fluid p-5 d-flex flex-column justify-content-center">
+            <p><span class="fs-5 fw-bold">2021</span><br>Penda conference in South Africa</p>
+          </div>
+        </div>
+
+        <div class="conference-container container-fluid text-light col-12 col-md-6" style="background-image: url(Images/recap2.jpg);">
+          <div class="conference-bg p-5 d-flex flex-column justify-content-center container-fluid ">
+            <p><span class="fs-5 fw-bold">2022</span><br>Penda conference in Dubai</p>
+          </div>        
+        </div>
+      </div>
+    </section>
+
+    <section class="sponsors container-fluid text-center pt-3">
+      <h2 class="fs-5 text-light ">Sponsors & Partners</h2>
+      <div class="line mt-2 mb-4"></div> 
+      <div class="all-sponsors container-fluid row overflow-auto py-4">
+        <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_3.png" alt="United Nations"></div>
+        <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_4.png" alt="Forbes"></div>      
+        <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_2.png" alt="Coca-cola company"></div>
+        <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_7.jpeg" alt="Glam"></div>
+        <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_ 17.png" alt="Reebok"></div>
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="Obama Foundation"></div>     
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_8.png" alt="Amazon"></div>
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_5.png" alt="Samsung"></div>
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_ 11.png" alt="sprite"></div>
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_ 13.png" alt="Samsung"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_6.png" alt="Fenty beauty"></div>     
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_9.png" alt="Microsoft"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_10.png" alt="IBM"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_ 14.png" alt="IBM"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="IBM"></div>
+        <a class="prev text-decoration-none d-none d-md-block">❮</a>
+        <a class="next text-decoration-none d-none d-md-block">❯</a>
+      </div>
+    </section>
```

Comment:
- [ ] As requested by the previou reviewer, https://github.com/c00p75/Capstone-project-Conference-page/pull/1#discussion_r1032726920 Please hide the partners section in mobile view for the homepage it should be present in the about page in mobile view see the design 
![image](https://user-images.githubusercontent.com/97521557/204079546-070705a3-4121-45e2-bdc8-6ce888f02c31.png)

And it is opposite for the desktop, you have to display partners setion in the home page and not in the about page see , Please see the[ design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) currently u have the partner section visible in bothhome and about page in both desktop and mobile view, I hope I was able clear thing up :pray: